src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableEntityContext.cs

```diff
@@ -124,6 +124,11 @@ internal void Writeback()
         void IDurableEntityContext.SignalEntity(EntityId entity, string operationName, object operationInput)
         {
             this.ThrowIfInvalidAccess();
+            if (operationName == null)
```

Comment:
Same question as in DurableCommonContext: do we want to allow whitespace/empty string ops?

Reply:
Again, same answer.